cmd/neofs-node/notificator.go

```diff
@@ -109,7 +109,7 @@ func (n notificationWriter) Notify(topic string, address oid.Address) {
 func initNotifications(c *cfg) {
 	if nodeconfig.Notification(c.appCfg).Enabled() {
 		topic := nodeconfig.Notification(c.appCfg).DefaultTopic()
-		pubKey := base58.Encode(c.cfgNodeInfo.localInfo.PublicKey())
+		pubKey := hex.EncodeToString(c.cfgNodeInfo.localInfo.PublicKey())
```

Comment:
Lets maybe unify the format by creating some function like
```
func StringifyNodeKey(netmap.NodeInfo) string {}
```
We can either place it to:
1. https://github.com/nspcc-dev/neofs-node/tree/master/pkg/core/netmap
2. https://github.com/nspcc-dev/neofs-sdk-go/tree/master/netmap

I also suggest to create key encoding function
```
func StringifyKeyBinary([]byte) string
```
and use it everywhere. It will allow to easily change string format if needed.

Reply:
Added usage of netmap.StringifyPublicKey().

Reply:
May be smth went wrong, I don't see change here